File: api/src/main/java/com/makingiants/today/api/repository/history/HistoryService.java
package com.makingiants.today.api.repository.history;

import com.makingiants.today.api.repository.history.pojo.Event;
import java.util.List;
import retrofit.http.GET;
import retrofit.http.Query;
import rx.Observable;

interface HistoryService {
  @GET("/main.json")
  Observable<List<Event>> get(@Query("day") int day, @Query("month") String month);
}

File: api/src/main/kotlin/com/makingiants/today/api/repository/history/HistoryService.kt
package com.makingiants.today.api.repository.history

import com.makingiants.today.api.repository.history.pojo.Event
import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Query

internal interface HistoryService {
  @GET("main.json")
  operator fun get(@Query("d") day: Int, @Query("m") month: Int): Observable<List<Event>>
}
